experimenter/experimenter/experiments/forms.py

```diff
@@ -0,0 +1,14 @@
+# forms.py
+
+from django import forms
+
+from .models import NimbusExperiment
```

Comment:
Ah, we usually use absolute imports everywhere.  I wonder if there's a lint rule I can add for that?

Reply:
yeah let me check that too